Assets/Spawner.cs

```diff
@@ -4,12 +4,70 @@
 
 public class Spawner : MonoBehaviour
 {
+    public GameObject PlatformPrefab;
+    public GameObject FinishPrefab;
     public GameObject CoinPrefab;
-    
+    float jump_length = 4.5f; // Approximate jump length given speed, jump speed, and gravitational constant derived from expiration.
```

Comment:
the naming scheme does not correspond to any commonly used code styles. C# does not use snake case at all. This variable should be named `jumpLength` or even `_jumpLength` (according to Microsoft's guidelines).
It is not required to use a specific coding convention, but you should at least be consistent with what most people do. It is important if you want to get extra points for code quality
https://docs.microsoft.com/ru-ru/dotnet/csharp/fundamentals/coding-style/coding-conventions